omar/openshift/templates/apps/omar-sqs-stager-app.json

```diff
@@ -67,6 +67,18 @@
       "displayName": "INSECURE_REPOSITORY",
       "description": "Mark the image stream as an insecure repository",
       "value": "false"
+    },
+    {
+      "name": "MEMORY_MAX",
+      "displayName": "MEMORY_MAX",
+      "description": "Maximum memory for this app to have",
+      "value": "1000Mi"
```

Comment:
Another higher default?

Reply:
These are just the basic defaults that get overwritten in the deployConfig.yml